components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/TopSitesUseCases.kt

```diff
@@ -23,6 +23,21 @@ class TopSitesUseCases(topSitesStorage: TopSitesStorage) {
         }
     }
 
+    /**
+     * Rename a top site use case.
+     */
+    class RenameTopSiteUseCase internal constructor(private val storage: TopSitesStorage) {
+        /**
+         * Renames the given [TopSite].
+         *
+         * @param topSite The top site.
+         * @param title The new title for the top site.
+         */
+        operator fun invoke(topSite: TopSite, title: String) {
+            storage.renameTopSite(topSite, title)
+        }
+    }
+
```

Comment:
Can we move this below RemoveTopSiteUseCase to keep the add/remove use case one after another?